src/rtkpos.c

```diff
@@ -2014,7 +2009,7 @@ static int relpos(rtk_t *rtk, const obsd_t *obs, int nu, int nr,
                 O H = partial derivatives
                 O R = double diff measurement error covariances
                 O vflg = list of sats used for dd  */
-        if ((nv=ddres(rtk,nav,obs,dt,xp,Pp,sat,y,e,azel,freq,iu,ir,ns,v,H,R,vflg))<4) {
+        if ((nv=ddres(rtk,obs,dt,rtk->x,rtk->P,sat,y,e,azel,freq,iu,ir,ns,v,H,R,vflg))<1) {
```

Comment:
nit: there are still a couple of reference to Pp in the comments just above and below, might it be clearer to update these from Pp to just P now.